test/unittest/models/unittest_models.cpp

```diff
@@ -892,9 +930,9 @@ GTEST_PARAMETER_TEST(
                  "multi_head_attention_float_attn_mask",
                  ModelTestOption::ALL_V2),
     /** @todo:change model if bool type tensor is supported */
-    mkModelTc_V2(makeMultiHeadAttention_float_attn_mask,
-                 "multi_head_attention_pseudo_bool_attn_mask",
-                 ModelTestOption::ALL_V2),
+    //  mkModelTc_V2(makeMultiHeadAttention_float_attn_mask,
```

Comment:
Could you add a comment explaining why this part is commented out?

Reply:
sure, thanks :)